concept/type/ThingType.java

```diff
@@ -35,6 +35,8 @@
 
     Stream<? extends Thing> getInstances();
 
+    long getInstancesCount();
```

Comment:
I wonder if this is really the right place for such a method. Do you know what the scope of statistics are? Presumably they will be used to optimise far more things than just retrieving instance counts, right?

If so I wonder if it warrants the creation of a new separate class, called say `ConceptStatistics` that is responsible for storing all statistics to do with concepts, and would include (for example) `instancesCount`, `ownsCount`, `linkedConceptCount` .... etc. Or is it just going to be `getInstancesCount` for the foreseeable future?

Reply:
I think this question should be raised to Haikal

Reply:
did Haikal respond?

Reply:
Not yet, I already sent him a discord message